src/styles/card.scss

```
@@ -66,14 +75,15 @@
 
     font-size: $middle-font-size;
     font-weight: $font-weight-bold;
-    text-transform: uppercase;
     line-height: 17px;
     color: $white-color;
-    text-decoration: none;
     background-color: $active-color;
     border: 1px solid $active-color;
     border-radius: 5px;
-    transition: background-color 3s;
+    text-decoration: none;
+    text-transform: uppercase;
+
+    transition: all 1s;
```

```suggestion
    transition: color 1s;
```
не рекомендується так робити, треба явно вказувати, які CSS правила анімуєш

---

Виправив, прописав для кожного значення окремо)